File: MyNotesProject/CommonLibrary/src/main/java/com/fsk/common/presentation/utils/animations/SimpleAnimatorListener.java
package com.fsk.common.presentation.utils.animations;


import android.animation.Animator;

/**
 * Animator Listener that defaults the Interface methods to be empty. This allows this classes
 * children to only implement the methods they want.
 */
public abstract class SimpleAnimatorListener implements Animator.AnimatorListener {


    @Override
    public void onAnimationStart(final Animator animation) {
        //Space Available For Rent
    }


    @Override
    public void onAnimationEnd(final Animator animation) {
        //Space Available For Rent
    }


    @Override
    public void onAnimationCancel(final Animator animation) {
        //Space Available For Rent
    }


    @Override
    public void onAnimationRepeat(final Animator animation) {
        //Space Available For Rent
    }
}

File: MyNotesProject/CommonLibrary/src/main/java/com/fsk/common/presentation/animations/SimpleAnimatorListener.kt
package com.fsk.common.presentation.animations

import android.animation.Animator

/**
 * Animator Listener that defaults the Interface methods to be empty. This allows this classes
 * children to only implement the methods they want.
 */
open class SimpleAnimatorListener : Animator.AnimatorListener {

    override fun onAnimationStart(animation: Animator) {
        //Space Available For Rent
    }


    override fun onAnimationEnd(animation: Animator) {
        //Space Available For Rent
    }


    override fun onAnimationCancel(animation: Animator) {
        //Space Available For Rent
    }


    override fun onAnimationRepeat(animation: Animator) {
        //Space Available For Rent
    }
}